test/mocha/browser/errors.spec.js

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+
```

Comment:
Related to https://github.com/bpmn-io/eslint-plugin-bpmn-io/pull/20/files#r1791477214

```suggestion
```